### PR TITLE
Fix warn and add a feature

### LINKED ISSUE #765: Warn not showing up for the ops
I've been using /warn for minor offenses lately, the warning shows up for the admin, not the op. This is a bug that I hope gets fixed. Let me look over the command.

### PR DESCRIPTION
**Pull request by Adaptified**
Fixes #765 and tells the player who warned them
@Wild1145

### CODE REVIEW

**Comment by Wild1145** [comment]
Looks like a good change to me.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java [hunk 1]
```diff
@@ -45,8 +45,10 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         }
 
         String warnReason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+        String admin = sender.getName();
 
-        playerMsg(ChatColor.RED + "[WARNING] " + warnReason);
+        player.sendMessage(ChatColor.RED + "[WARNING] " + warnReason);
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: Please use playerMsg(CommandSender, String) here.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java [hunk 2]
```diff
@@ -45,8 +45,10 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         }
 
         String warnReason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+        String admin = sender.getName();
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_warn.java]: Unnecessary use of a variable here, it's only used once.

**Comment by Wild1145** [comment]
@JeromSar To Re-Open?

**Comment by Adaptified** [comment]
No, new issue opened, branch deleted. @JeromSar

**Pull request closed without merging**